Cubical/Categories/Limits/BinProduct/More.agda

```diff
@@ -140,25 +174,63 @@ module _ (C : Category ℓ ℓ') where
       ×-extensionality : π₁ ∘⟨ C ⟩ f ≡
                          π₁ ∘⟨ C ⟩ g → π₂ ∘⟨ C ⟩ f ≡ π₂ ∘⟨ C ⟩ g → f ≡ g
       ×-extensionality p1 p2 = extensionality (ues _) (ΣPathP (p1 , p2))
-  module Notation (bp : BinProducts C) where
+
+  module NotationAt {a b : C .ob} (bp : BinProduct C a b) where
```

Comment:
Moved much of Notation into NotationAt as possible, except for things that require all products like the bifunctoriality of (- × -) : C × C → C